node/impl/client/client.go

```diff
@@ -400,7 +400,13 @@ func (a *API) ClientListImports(ctx context.Context) ([]api.Import, error) {
 	return out, nil
 }
 
-func (a *API) ClientRetrieve(ctx context.Context, order api.RetrievalOrder, ref *api.FileRef) (<-chan marketevents.RetrievalEvent, error) {
+func (a *API) ClientRetrieve(ctx context.Context, order api.RetrievalOrder, ref *api.FileRef) error {
+	events := make(chan marketevents.RetrievalEvent)
```

Comment:
Probably not create the channel and populate it as in the event-ful case, but 🤷‍♀️

Reply:
Made it blocking / behave as it did previously (I think)